config.go

```diff
@@ -215,6 +215,7 @@ func getConfig() *types.Configuration {
 	v.SetDefault("PolicyReport.Kubeconfig", "")
 	v.SetDefault("PolicyReport.MinimumPriority", "")
 	v.SetDefault("PolicyReport.Warning", 4)
+	v.SetDefault("PolicyReport.MaxReports", 10)
```

Comment:
keep in mind to update README.md and config_example.yaml too, it might help who would like to test your branch